### DISCUSSION TITLE
Issue comments UI jumps scroll state

### DISCUSSION THREAD

**Discussion by crackwitz** [community/community#117123] [Mobile]
I scroll through comments. Let's say I go from bottom to top. The comments appear to be instantiated on the fly. As I scroll up, the comment I've looked at jumps down because the one above it changes height and displaces that one.

**Comment by eliperkins** [comment]
Hey @crackwitz , thanks for this feedback! I'd love to get some more info for ya to help fix this frustrating scrolling bug.

- What OS and device are you running GitHub Mobile on? (e.g., iOS 17.4 on iPhone 15, Android 14 on Pixel)
- Do you have a link to an issue that is able to consistently reproduce this bug? Having a test case to bring to our engineering team will help it get solved even faster!

Thanks for helping us improve the quality of GitHub Mobile 🙏
